index.css

```diff
@@ -0,0 +1,318 @@
+body{
+    margin: 0;
+    padding: 0;
+    width: 100vw;
+    overflow-x: hidden;
+    /* position: relative; */
+    
+}
+#header{
+    margin: 0;
+    /* position: sticky; */
+    padding: 10px 20px;
+    /* background-color: aqua; */
+    display: flex;
+    width: 100vw;
+    /* overflow-x: hidden; */
+    /* align-items: center;
+    justify-content: center; */
```

Comment:
kindly remove all commented out unused code in your code base. Comments of unused code negative affects the time space complexities of your code. since it provides no relevance, I think its best to remove them for cleaner code and better reflections in time space complexities

Reply:
- [ ] I'm afraid commented lines of code have something to do with the `time and space complexity` problem. Kinldy you may consider giving a relevant `why` of changes and if you do not know you may look for it on the internet since we learn by reviewing someone else code :smile: